### PR TITLE
Continuation of @jschuelller's 5.12.4 PR.

### PR DESCRIPTION
**Pull request by mingwandroid**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ocefpaf** [comment]
@jschueller and @mingwandroid please let me know when O should try to build these locally for the upload.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -61,7 +78,8 @@ source:
     folder: opengl32sw                                                                                            # [win64]
 
 build:
-  number: 0
+  number: {{ build_number }}
+  string: {{ build_number }}{{ dev_nature }}
```
Comment (isuruf) [inline recipe/meta.yaml]: Use the pkg hash here.

File: recipe/conda_build_config.yaml [hunk 2]
```diff
@@ -1,2 +1,4 @@
+CONDA_BUILD_SYSROOT:       # [osx]
+  - /opt/MacOSX10.12.sdk   # [osx]
```
Comment (isuruf) [inline recipe/conda_build_config.yaml]: Remove these 2 lines

Comment (mingwandroid) [reply recipe/conda_build_config.yaml]: We need these for Anaconda Distro, are they harmful?

File: recipe/meta.yaml (hunk 1 above)
Comment (mingwandroid) [reply recipe/meta.yaml]: For my debugging purposes I need to switch this depending on `dev_nature`.

Comment (mingwandroid) [reply recipe/meta.yaml]: i.e `{% if dev_nature != '' %}` .. around it or something like that. `dev_nature` will not be set unless forced via the command line.

File: recipe/conda_build_config.yaml (hunk 2 above)
Comment (isuruf) [reply recipe/conda_build_config.yaml]: Yes, it's harmful for conda-forge as we are installing the SDK somewhere else.

File: recipe/meta.yaml (hunk 1 above)
Comment (isuruf) [reply recipe/meta.yaml]: I meant use the pkg hash in addition. Something like `{{ PKG_HASH }}_{{ build_number }}{{ dev_nature }}`

File: recipe/conda_build_config.yaml (hunk 2 above)
Comment (mingwandroid) [reply recipe/conda_build_config.yaml]: OK, I'm probably OK with aligning our location then. We do need to force 10.12 somehow though. What is the default ATM? Any clues how to force 10.12?

File: recipe/meta.yaml (hunk 1 above)
Comment (mingwandroid) [reply recipe/meta.yaml]: OK, cool

File: recipe/conda_build_config.yaml (hunk 2 above)
Comment (isuruf) [reply recipe/conda_build_config.yaml]: It is set to `$(xcode-select -p)/Platforms/MacOSX.platform/Developer/SDKs/MacOSX${MACOSX_DEPLOYMENT_TARGET}.sdk` which wade sense when we were using system compilers.

Default is 10.9. If you set MACOSX_DEPLOYMENT_TARGET to 10.12 as you've done here in `recipe/conda_build_config.yaml`, the CI script will read that variable, download 10.12, extract it and automatically set `CONDA_BUILD_SYSROOT`. So there's no need to set `CONDA_BUILD_SYSROOT` in the recipe. Just `MACOSX_DEPLOYMENT_TARGET` would do.

**Comment by jschueller** [comment]
@isuruf, your commit fails:
```
cp: ../licenses/LICENSE.FDL: No such file or directory
```

**Comment by ccordoba12** [comment]
Sure, I think so too.

**Pull request closed without merging**